clients/web/test/specRunner.js

```diff
@@ -78,20 +75,14 @@ page.onConsoleMessage = function (msg) {
             }));
         }
         return;
-    }
-
-    if (accumCoverage && coverageOutput) {
-        try {
-            fs.write(coverageOutput, msg, 'a');
-        } catch (e) {
-            console.log('Exception writing coverage results: ', e);
+    } else if (msg === 'ConsoleReporter finished') {
+        var success = this.page.evaluate(function () {
+            return window.jasmine_phantom_reporter.status === 'success';
+        });
+        if (success) {
+            reportCoverage();
         }
-    } else {
-        console.log(msg);
```

Comment:
I didn't notice it before, I can move it after the conditional so the `__SCREENSHOT__` lines are not shown.